wireless_modules/test.py

```diff
@@ -11,11 +11,16 @@
     default=2, help="""The module number to determine the publish and subscription topics. 
     If nothing is selected it will send on module number 2 topics.""")
 parser.add_argument(
-    '--host', action='store', type=str, default="5.196.95.208",
+    '--host', action='store', type=str, default="localhost",
     help="""Address of the MQTT broker. If nothing is selected it will
     default to 5.196.95.208 - a test server by mosquito.""")
 
 
+def on_connect(client, data, message):
```

Comment:
Should this be called `on_message`? Also, I don't believe returning `True` means anything to `paho-mqtt`.